doc/deploying/controlling-with-url-parameters.md

```diff
@@ -48,34 +53,12 @@ The `start=` parameter essentially embeds an entire catalog file in the URL. The
             ]
         },
         {
-            "initialCamera": {
```

Comment:
Can you not set initialCamera any more?

Reply:
You can still set `initialCamera` - I just thought it is cluttering up the doc. Full properties supported is listed in `ShareData` link above

https://github.com/TerriaJS/terriajs/blob/d4dbbadf34054b512d98d5608c62b30c56294f6b/doc/deploying/controlling-with-url-parameters.md#L39

Reply:
Sounds good.